how-to-guides/installation.ipynb

```diff
@@ -0,0 +1,102 @@
+    "Hub](https://hub.docker.com/u/pytorchignite) using :\n",
+    "\n",
+    "``` shell\n",
+    "docker pull IMAGE_NAME\n",
+    "```\n",
+    "\n",
+    "Available pre-built images are :\n",
+    "\n",
+    "|               |                                           Base                                           |                                             Horovod                                              |                                            MS DeepSpeed                                            |\n",
+    "|:-------------:|:----------------------------------------------------------------------------------------:|:------------------------------------------------------------------------------------------------:|:--------------------------------------------------------------------------------------------------:|\n",
+    "|     Base      |        [`pytorchignite/base:latest`](https://hub.docker.com/r/pytorchignite/base)        |        [`pytorchignite/hvd-base:latest`](https://hub.docker.com/r/pytorchignite/hvd-base)        |                                                 X                                                  |\n",
+    "|    Vision     |      [`pytorchignite/vision:latest`](https://hub.docker.com/r/pytorchignite/vision)      |      [`pytorchignite/hvd-vision:latest`](https://hub.docker.com/r/pytorchignite/hvd-vision)      |                                                 X                                                  |\n",
+    "|      NLP      |         [`pytorchignite/nlp:latest`](https://hub.docker.com/r/pytorchignite/nlp)         |         [`pytorchignite/hvd-nlp:latest`](https://hub.docker.com/r/pytorchignite/hvd-nlp)         |                                                 X                                                  |\n",
+    "|  NVIDIA Apex  |        [`pytorchignite/apex:latest`](https://hub.docker.com/r/pytorchignite/apex)        |        [`pytorchignite/hvd-apex:latest`](https://hub.docker.com/r/pytorchignite/hvd-apex)        |        [`pytorchignite/msdp-apex:latest`](https://hub.docker.com/r/pytorchignite/msdp-apex)        |\n",
+    "| Apex + Vision | [`pytorchignite/apex-vision:latest`](https://hub.docker.com/r/pytorchignite/apex-vision) | [`pytorchignite/hvd-apex-vision:latest`](https://hub.docker.com/r/pytorchignite/hvd-apex-vision) | [`pytorchignite/msdp-apex-vision:latest`](https://hub.docker.com/r/pytorchignite/msdp-apex-vision) |\n",
+    "|  Apex + NLP   |    [`pytorchignite/apex-nlp:latest`](https://hub.docker.com/r/pytorchignite/apex-nlp)    |    [`pytorchignite/hvd-apex-nlp:latest`](https://hub.docker.com/r/pytorchignite/hvd-apex-nlp)    |    [`pytorchignite/msdp-apex-nlp:latest`](https://hub.docker.com/r/pytorchignite/msdp-apex-nlp)    |\n",
+    "\n",
+    "and run it with Docker v19.03+ :\n",
+    "\n",
+    "``` shell\n",
+    "docker run --gpus all -it -v $PWD:/workspace/project --network=host --shm-size 16G IMAGE_NAME\n",
```

Comment:
What do you think to add an example to run one of the usecase of the database ? (cifar, voc, whatever)

Reply:
Do you mean a tutorial on pulling a docker image and then training a basic model on any popular dataset?

Reply:
Yes, using an example installed in the images.